app/views/posts/_posts.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class='col-9 caja1 mx-auto caja'>    
+    <% @posts.each do |post| %>
+        
+            <div class='col-11  bg-light row mx-auto'> 
+                <% if usuario_signed_in?%>          
+                    <div class='col-2 mx-auto bg-light'>                        
+                        <div>
+                            <p class="image is-64x64 ">
+                                
+                                <img src="https://bulma.io/images/placeholders/64x64.png" class='img-container'>
+                            </p>
+                        </div>
+                            <strong><p> <%= post.usuario.name %></p></strong><br />                      
+                    </div>
+                <% end %>    
```

Comment:
- [ ] Please put the contents of this if statement in a partial and move the if statement to a helper method that renders the partial and call the helper method here instead of having the logic directly in this view